src/components/MessagesList/MessagesGroup/Message/MessagePart/MessageBody.vue

```diff
@@ -432,16 +430,33 @@ export default {
 		showJoinCallButton() {
 			EventBus.emit('scroll-chat-to-bottom', { smooth: true })
 		},
+
+		containsCodeBlocks(value) {
+			if (value) {
+				this.$refs.messageMain.addEventListener('mouseover', this.handleMarkdownMouseOver)
+				this.$refs.messageMain.addEventListener('mouseleave', this.handleMarkdownMouseLeave)
+			} else {
+				this.$refs.messageMain.removeEventListener('mouseover', this.handleMarkdownMouseOver)
+				this.$refs.messageMain.removeEventListener('mouseleave', this.handleMarkdownMouseLeave)
+			}
+		},
```

Comment:
This is quite not a Vue way. Does it really have a noticeable performance impact?
If it does, I'd propose to add the copy button directly to the code block.